init_db.py

```
@@ -1,83 +1,66 @@
+
+# Convert asyncpg → psycopg2 for Alembic if needed
+if db_url.startswith("postgresql+asyncpg://"):
+    sync_url = db_url.replace("postgresql+asyncpg://", "postgresql+psycopg2://", 1)
+else:
+    sync_url = db_url
+
+# --- Run Alembic
+try:
+    from alembic import command
+    from alembic.config import Config
+except Exception as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)